[2025-01-04T15:31:32.744+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.ft_posting_f manual__2025-01-04T15:21:18.190378+00:00 [queued]>
[2025-01-04T15:31:32.756+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.ft_posting_f manual__2025-01-04T15:21:18.190378+00:00 [queued]>
[2025-01-04T15:31:32.757+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 3
[2025-01-04T15:31:32.774+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ft_posting_f> on 2025-01-04 15:21:18.190378+00:00
[2025-01-04T15:31:32.782+0000] {standard_task_runner.py:60} INFO - Started process 168 to run task
[2025-01-04T15:31:32.786+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'ft_posting_f', 'manual__2025-01-04T15:21:18.190378+00:00', '--job-id', '1872', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmp2urodk26']
[2025-01-04T15:31:32.789+0000] {standard_task_runner.py:88} INFO - Job 1872: Subtask ft_posting_f
[2025-01-04T15:31:33.031+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.ft_posting_f manual__2025-01-04T15:21:18.190378+00:00 [running]> on host 00bb408c4f3b
[2025-01-04T15:31:33.121+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abershits' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='ft_posting_f' AIRFLOW_CTX_EXECUTION_DATE='2025-01-04T15:21:18.190378+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-04T15:21:18.190378+00:00'
[2025-01-04T15:31:38.174+0000] {base.py:83} INFO - Using connection ID 'postgres_db' for task execution.
[2025-01-04T15:32:03.055+0000] {base.py:83} INFO - Using connection ID 'postgres_db' for task execution.
[2025-01-04T15:32:03.076+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-04T15:32:03.087+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=ft_posting_f, execution_date=20250104T152118, start_date=20250104T153132, end_date=20250104T153203
[2025-01-04T15:32:03.117+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-04T15:32:03.141+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
